src/Core/Models/Table/EmergencyAccess.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using Bit.Core.Enums;
+using Bit.Core.Utilities;
+
+namespace Bit.Core.Models.Table
+{
+    public class EmergencyAccess : ITableObject<Guid>
+    {
+        public Guid Id { get; set; }
+        public Guid GrantorId { get; set; }
+        public Guid? GranteeId { get; set; }
+        public string Email { get; set; }
+        public string KeyEncrypted { get; set; }
+        public EmergencyAccessType Type { get; set; }
+        public EmergencyAccessStatusType Status { get; set; }
+        public int WaitTimeDays { get; set; }
+        public DateTime? RecoveryInitiatedAt { get; internal set; }
+        public DateTime? LastNotificationAt { get; internal set; }
```

Comment:
These should follow the naming convention of all DateTime fields:

```suggestion
        public DateTime? RecoveryInitiatedDate { get; internal set; }
        public DateTime? LastNotificationDate { get; internal set; }
```